components/match2/wikis/leagueoflegends/legacy/match_group_legacy_default.lua

```diff
@@ -6,36 +6,17 @@
 -- Please see https://github.com/Liquipedia/Lua-Modules to contribute
 --
 
-local MatchGroupLegacyDefault = {}
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
 
-local String = require('Module:StringUtils')
-local Logic = require('Module:Logic')
+local MatchGroupLegacy = Lua.import('Module:MatchGroup/Legacy')
 
-local MAX_NUM_MAPS = 9
+---@class LoLMatchGroupLegacyDefault: MatchGroupLegacy
+local LoLMatchGroupLegacyDefault = Class.new(MatchGroupLegacy)
```

Comment:
only the class in annotations has to be different ;)

Reply:
:|, I'm noob